### DISCUSSION TITLE
ldap integration

### DISCUSSION THREAD

**Discussion by AhmedAbuElyazeed** [apache/airflow#31152] [Q&A]
Greetings,
I'm trying to integrate airflow with ldap server.
I made sure that airflow can communicate with ldap server and requested the configurations from ldap team, but whenever I try to login, the web portal shows "Invalid login, please try again" message.
How can I check the logs for flask_appbuilder? to troubleshoot this, or what can I try?
___this is my webserver_config.py___
from future import annotations
import os
from airflow.[www.fab_security.manager](http://www.fab_security.manager/) import AUTH_LDAP
basedir = os.path.abspath(os.path.dirname(__file__))
Flask-WTF flag for CSRF
WTF_CSRF_ENABLED = True
WTF_CSRF_TIME_LIMIT = None
AUTH_TYPE = AUTH_LDAP
AUTH_ROLE_ADMIN = 'Admin'
AUTH_USER_REGISTRATION = True
AUTH_USER_REGISTRATION_ROLE = "Viewer"
AUTH_LDAP_SERVER = 'ldap://myLDAPServer:389'
AUTH_LDAP_SEARCH = 'OU=users,DC=x,DC=y'
AUTH_LDAP_BIND_USER = 'CN=bind_user,OU=services,DC=x,DC=y'
AUTH_LDAP_BIND_PASSWORD = '*********'
AUTH_LDAP_UID_FIELD = 'CN'
AUTH_LDAP_USERNAME_FORMAT = 'CN=%s,OU=services,DC=x,DC=y'
LDAPS
AUTH_LDAP_USE_TLS = False
AUTH_LDAP_ALLOW_SELF_SIGNED = False
AUTH_LDAP_TLS_CACERTFILE = '/etc/ssl/certs/ldap.crt'

**Comment by DevFontes** [comment]
Were you able to authenticate?

I'm using the chart https://github.com/bitnami/charts/tree/main/bitnami/airflow

Kubernetes Cluster 1.28

The values are this:

ldap:
   enabled: true
   uri: "ldap://10.1.1.2:389"
   basedn: "dc=**,dc=**,dc=**,dc=br"
   searchAttribute: "cn"
   binddn: "user@**.**.**.br"
   bindpw: "password"
   userRegistration: 'True'
   userRegistrationRole: "Admin"
   rolesMapping: '{ "cn=All,ou=Groups,dc=**,dc=**,dc=***,dc=br": ["User"], "cn=group_name,ou=Groups,dc =**,dc=**,dc=***,dc=br": ["Admin"], }'
   rolesSyncAtLogin: 'True'
   group_member_attr: "memberOf"
   superuser_filter: "memberOf=CN=group_name,OU=Groups,OU=****,DC=**,DC=**,DC=***,DC=br"
   data_profiler_filter: "memberOf=CN=group_name,OU=Group,OU=****,DC=**,DC=**,DC=***,DC=br"
   search_scope: "SUBTREE"
   user_filter: "ou=users,dc=**,DC=**,DC=***,DC=br"
   user_name_attr: "sAMAccountName"

Some users log in, with others I get the error:

Invalid login. Please try again.
